ios/ReactTestApp.xcodeproj/project.pbxproj

```diff
@@ -403,7 +403,7 @@
 				CODE_SIGN_ENTITLEMENTS = App.entitlements;
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_ASSET_PATHS = "";
-				DEVELOPMENT_TEAM = UBF8T346G9;
+				DEVELOPMENT_TEAM = "";
```

Comment:
Can we remove these lines instead? The macOS project doesn't have them, so it should be fine.

Reply:
Yeah makes sense, removed